target/linux/ramips/image/mt7621.mk

```diff
@@ -544,6 +544,23 @@ define Device/linksys_ea7500-v2
 endef
 TARGET_DEVICES += linksys_ea7500-v2
 
+define Device/linksys_ea7300
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 36864k
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := EA7300
```

Comment:
How probable is it that there will be a v3 (with different setup)?

Having linksys_ea7300 and linksys_ea7300-v3 would not be so helpful, then I'd consider to split and explicitly define ea7300-v1 and -v2 already.

Reply:
You're right, I separated both versions into their own devices.